modules/distr-tracing-otel-config-collector.adoc

```diff
@@ -331,6 +331,238 @@ rules:
         metrics:
           processors: [resourcedetection]
 ----
+==== Attributes Processor
+
+The Attributes Processor has the ability to modifying attributes of a span, log, or metric. This processor can be configuret to filter and match input data and determine whether they should be included or excluded for specific actions.
+
+The processor operates on a list of actions, executing them in the order specified in the configuration. The following actions are supported:
+
+* Insert: Inserts a new attribute into the input data when the specified key does not already exist.
+* Update: Updates an attribute in the input data if the key already exists.
+* Upsert: Combines the insert and update actions. It inserts a new attribute if the key doesn't exist, and updates the attribute if the key does exist.
+* Delete: Removes an attribute from the input data.
+* Hash: Performs a SHA1 hash on an existing attribute value.
```

Comment:
I'm struggling to understand what "performs a SHA1" mean. Does it add a SHA1 or check a SHA1?